lib/kernel-manager.coffee

```diff
@@ -77,6 +78,13 @@ class KernelManager
                 )
                 finishKernelStartup kernel
 
+    attachToGatewaySession: (grammar, session, onAttach) ->
```

Comment:
Rename to `startWSKernelFor` (to be consistent with the existing naming of `startKernel` and `startKernelFor`).